services/common/src/redux/customAxios.js

```diff
@@ -3,9 +3,9 @@ import { notification, Button } from "antd";
 import * as String from "@mds/common/constants/strings";
 import React from "react";
 import * as API from "@mds/common/constants/API";
-import { ENVIRONMENT } from "@mds/common";
+import { ENVIRONMENT } from "@mds/common/constants";
```

Comment:
it's a bit funny to me to see your one-man mission to make imports better. 😂 
Keep fighting the good fight!

Reply:
Haha, my IDE complains about it, and I hate seeing red lines

Reply:
ahhhh mine doesn't, so I assumed it was preference. But mine did start complaining about unused imports in python recently, which I have been enjoying. 🪓

Reply:
It is satisfying to do these little cleanups.  With the common repo there's also the fact that there are many variables with identical names in different folders (legacy from before the common repo).  It all seems to mostly work, but it has the potential to break without more specific imports.